### DISCUSSION TITLE
Why doesn't it have a hosted Helm chart?

### DISCUSSION THREAD

**Discussion by hakuno** [requarks/wiki#4143] [Help / Questions]
Why doesn't it have a hosted Helm chart?

I had to apply the Helm Chart by using thirdy part software, but I found no URL to input.

See more https://helm.sh/docs/topics/chart_repository/#github-pages-example

Does it have any or any plan to do?

**Comment by Grafnus** [comment]
As far as I can tell it looks like it will be a feature in the upcoming wiki.js 3.0 release:

https://github.com/Requarks/wiki/tree/dev/dev/helm

But I havn't heard of it in the blog posts yet.
